packages/dataclass-editor/src/components/dataclass/master/DataClassMaster.tsx

```diff
@@ -0,0 +1,101 @@
+import {
+  BasicField,
+  Button,
+  SelectRow,
+  Table,
+  TableBody,
+  TableCell,
+  TableResizableHeader,
+  useReadonly,
+  useTableSelect
+} from '@axonivy/ui-components';
+import { IvyIcons } from '@axonivy/ui-icons';
+import { flexRender, getCoreRowModel, useReactTable, type ColumnDef } from '@tanstack/react-table';
+import { isRowSelected, selectRow } from '../../../utils/table';
+import { Control } from '../../control/Control';
+import {
+  dataClassFieldCommentAttribute,
+  dataClassFieldNameAttribute,
+  dataClassFieldTypeAttribute,
+  type DataClassField
+} from '../data/dataclass';
+import { isPersistent } from '../data/dataclass-field-utils';
+import './DataClassMaster.css';
+
+type DataClassMasterProps = {
+  dataClassFields: Array<DataClassField>;
+};
+
+export const DataClassMaster = ({ dataClassFields }: DataClassMasterProps) => {
+  const selection = useTableSelect<DataClassField>();
+  const columns: Array<ColumnDef<DataClassField, string>> = [
+    {
+      accessorKey: dataClassFieldNameAttribute,
```

Comment:
I'm definilty against this strange abstraction of this field name!
Why not simply use 'name'?
- I mean do we expect this to change somewhen in the near future? No
- Don't we write tests which tell us that something is not correct anymore? Yes

If you don't like this soft accessorkey then why not simply use the accessorFn which is typesafe?